modules/launcher/src/main/java/org/mule/module/launcher/artifact/Artifact.java

```diff
@@ -71,4 +72,11 @@
      * @return MuleContext created from the artifact configurations files.
      */
     MuleContext getMuleContext();
+    
+    /**
+     * Sets the configuration management properties
+     * 
+     * @param configurationManagementProperties
+     */
+    void setConfigurationManagementProperties(Properties configurationManagementProperties);
```

Comment:
Let's call this concept "Deployment properties" instead of "Configuration management" all over the codde